torch_uncertainty/models/resnet/std.py

```diff
@@ -197,11 +199,15 @@ def __init__(
         dropout_rate: float,
         groups: int,
         style: str = "imagenet",
+        num_estimators: int = None,
+        enable_last_layer_dropout: bool = False,
```

Comment:
I'll change this variable to last__layer_dropout or ll_dropout. @alafage, the latter is not explicit enough?